common/metrics/interfaces.go

```diff
@@ -73,3 +73,10 @@ type (
 		Tagged(tags ...Tag) Scope
 	}
 )
+
+var sanitizer = tally.NewSanitizer(tally.SanitizeOptions{
```

Comment:
All metrics seems to use underscore already. I don't understand why they/we need this.